release_notes/372-release-notes.adoc

```diff
@@ -8,4 +8,262 @@ toc::[]
+
+[id="network-graph-improvements"]
+=== Network graph improvements
+
+{product-title-short} 3.72 includes the following improvements to the Network Graph:
+
+* Before this improvement, the Legend in the Network Graph required hovering over the symbols representing namespaces, deployments, and connections. The updated legend shows the symbols and explanatory text together. 
+* Sometimes, the graph view edges did not connect the edges of nodes and overlapped on namespaces. This issue is fixed.
+* Before this improvement, there were differences in the YAML file viewer instances. With this update, {product-title-short} shows the same formatting and interface when you view YAML files. 
+
+[id="doc-updates-372"]
+=== Documentation updates
+
+The {product-title-short} documentation has been updated to include a xref:../operating/default-security-policies.adoc#default-security-policies[list of default security policies].
+
+[id="notable-tech-changes-372"]
+== Notable technical changes
+
+[id="rhel9-scan"]
+=== Scanning support for Red Hat Enterprise Linux 9
+
+{product-title-short} 3.72 introduces support for analyzing images built with Red Hat Universal Base Image (UBI) 9 and Red Hat Enterprise Linux (RHEL) 9 RPMs for vulnerabilities.
+
+RHEL 9 is now generally available (GA). RHEL 9 can run your code efficiently whether deployed on physical infrastructure, in a virtual machine, or in containers built from Red Hat UBIs. {product-title-short} supports RHEL 9 scanning with this release. 
+
+[id="fixable-cvss6-disabled"]
+=== Policy for CVEs with fixable CVSS of 6 or greater disabled by default 
+
+Beginning with this release, the `Fixable CVSS >= 6 and Privileged` policy is no longer enabled by default for new {product-title-short} installations. The configuration of this policy is not changed when upgrading an existing system. For new installations, this policy is replaced by the policy `Privileged Containers with Important and Critical Fixable CVEs`, which gives an alert for containers running in privileged mode that have important or critical fixable vulnerabilities. The new policy is also in the `Deploy` life cycle. Known vulnerabilities make it easier for adversaries to exploit your application, and highly-privileged containers pose greater risk. You can fix these high-severity vulnerabilities by updating to a newer version of the affected components or running your container with lower privileges.
```

Comment:
```suggestion
Beginning with this release, the `Fixable CVSS >= 6 and Privileged` policy is no longer enabled by default for new {product-title-short} installations. The configuration of this policy is not changed when upgrading an existing system. A new policy `Privileged Containers with Important and Critical Fixable CVEs`, which gives an alert for containers running in privileged mode that have important or critical fixable vulnerabilities, has now been added. The new policy is also in the `Deploy` life cycle. Known vulnerabilities make it easier for adversaries to exploit your application, and highly-privileged containers pose greater risk. You can fix these high-severity vulnerabilities by updating to a newer version of the affected components or running your container with lower privileges.
```

Maybe? To clarify that the new policy is added for both new and existing installations. Only the disabling is for new installations only.